mix.exs

```diff
@@ -88,6 +90,7 @@ defmodule Hexpm.MixProject do
     project = Mix.Project.config()
     proto_paths = project[:proto_paths] || ["priv/proto"]
     erlc_path = project[:erlc_paths] |> List.first()
+    File.mkdir_p!(erlc_path)
```

Comment:
@ericmj ok, this is a cleaner solution than having `src/.gitkeep`. Obviously we will need neither gpb nor `src/` at all in hexpm when hex_erl is finished!